index.html

```diff
@@ -243,7 +248,25 @@ <h3>LET’S CONNECT</h3>
         </div>
       </section>
       <!--End of about section-->
-
-
+       <!--Start of Contact Page-->
+       <section class="contact-page">
+        <div class="contact-block">
+            <div class="description">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                    I’d love to help with it!</p>
+            </div>
+            <form action="#">
+                <fieldset>
+                    <input type="text" placeholder="Lorem Ipsum">
+                    <input type="email" placeholder="loremipsum@gmail.com">
+                    <textarea name="" id="" cols="30" rows="10" placeholder="Write your message here..."></textarea>
+                </fieldset>
```

Comment:
- [ ] It's requested message fields accepts **up to 500 chars**, Therefore I suggest you add `maxlength="500"` attribute to the message input.
- [ ] As pointed earlier :point_up: Kindly add `required` attribute to this `textarea`